logstash-core/src/main/java/org/logstash/config/ir/compiler/ComputeStepSyntaxElement.java

```diff
@@ -51,6 +52,10 @@
 
     private static final ISimpleCompiler COMPILER = new SimpleCompiler();
 
+    static {
+        COMPILER.setDebuggingInformation(true, true, true);
+    }
+
```

Comment:
I don't believe this was intended to stay?

```suggestion
```

Reply:
I was thinking the same, but if the switch for Janino debug is on, probably having the right line reference on the generated sources could be helpful.

Reply:
Do you think could impact negatively the compilation part?

Reply:
I haven't tested. can't this be enabled through JAVA_OPTS?

Reply:
I found it, it's `org.codehaus.janino.source_debugging.enable` so removed the code fragment